manim/mobject/graph.py

```diff
@@ -851,7 +842,7 @@ def _create_vertices(
                 label_fill_color=label_fill_color,
                 vertex_type=vertex_type,
                 vertex_config=vertex_config[v],
-                vertex_mobject=vertex_mobjects[v] if v in vertex_mobjects else None,
+                vertex_mobject=vertex_mobjects.get(v, None),
```

Comment:
Can't we just use `get` here?

```suggestion
                vertex_mobject=vertex_mobjects.get(v),
```